File: app/src/main/java/org/mifos/selfserviceapp/models/Currency.java
package org.mifos.selfserviceapp.models;

/**
 * Created by michaelsosnick on 12/11/16.
 */

public class Currency {
    private String code;
    private String name;
    private int decimalPlaces;
    private String displaySymbol;
    private String nameCode;
    private String displayLabel;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public int getDecimalPlaces() {
        return decimalPlaces;
    }

    public void setDecimalPlaces(int decimalPlaces) {
        this.decimalPlaces = decimalPlaces;
    }

    public String getDisplaySymbol() {
        return displaySymbol;
    }

    public void setDisplaySymbol(String displaySymbol) {
        this.displaySymbol = displaySymbol;
    }

    public String getNameCode() {
        return nameCode;
    }

    public void setNameCode(String nameCode) {
        this.nameCode = nameCode;
    }

    public String getDisplayLabel() {
        return displayLabel;
    }

    public void setDisplayLabel(String displayLabel) {
        this.displayLabel = displayLabel;
    }
}

File: app/src/main/java/org/mifos/mobilebanking/models/Currency.kt
package org.mifos.mobilebanking.models

import android.os.Parcelable
import kotlinx.android.parcel.Parcelize

/**
 * Created by michaelsosnick on 12/11/16.
 */

@Parcelize
data class Currency(
        var code: String? = null,
        var name: String? = null,
        var decimalPlaces: Int = 0,
        var displaySymbol: String? = null,
        var nameCode: String? = null,
        var displayLabel: String? = null


) : Parcelable